src/mbgl/renderer/painter.cpp

```diff
@@ -327,6 +327,7 @@ void Painter::renderPass(PaintParameters& parameters,
 
             extrusionTexture->bind();
 
+            context.scissorTest = false;
```

Comment:
Instead of setting this here, we should move that call into OffscreenTexture/OffscreenView's `bind` method, which is also where we call `context.viewport = { 0, 0, size };`